CHANGELOG.md

```diff
@@ -0,0 +1,97 @@
+# Documenter.jl changelog
+
+## Version `v0.20-dev`
+
+* Documenter now requires at least Julia v0.7. ([#795][github-795])
+
+* ![BREAKING][badge-breaking] Documentation deployment via the `deploydocs` function has changed considerably.
+
+  - The user-facing directories (URLs) of different versions and what gets displayed in the version selector have changed. By default, Documenter now creates the `stable/` directory (as before) and a directory for every minor version (`vX.Y/`). The `release-X.Y` directories are no longer created. ([#706][github-706], [#813][github-813], [#817][github-817])
+
+    Technically, Documenter now deploys actual files only to `dev/` and `vX.Y.Z/` directories. The directories (URLs) that change from version to version (e.g. `latest/`, `vX.Y`) are implemented as symlinks on the `gh-pages` branch.
+
+    The version selector will only display `dev/`, `stable/` and `vX.Y/` directories by default. This behavior can be customized with the `selector` keyword of `deploydocs`.
```

Comment:
"keyword argument" maybe?

Reply:
I consciously standardised here to "keyword", since that is what the docs have been using since the beginning (before my time; see e.g. the `makedocs`' docstring).

I believe it is clear what is meant and having to write "keyword argument" everywhere quickly gets old. Also, while technically function calls, `make.jl` is a bit like a configuration file, so I'd say we're allowed to invent our own terminology here.